modules/adding-to-a-project.adoc

```diff
@@ -15,3 +15,8 @@ navigation menu.
 . At the top of the page, select the name of the project that you want to add to.
 
 . Click on a method for adding to your project, and then follow the workflow.
```

Comment:
```suggestion
. Click a method for adding to your project, and then follow the workflow.
```
What is the user adding to their project? This phrase uses "add" as a transitive verb. For example, "You are adding something to something else." 

Perhaps add a brief abstract to line 7.